HarClient.Tests/Validation/HarAssert.cs

```diff
@@ -0,0 +1,251 @@
+﻿using HarSharp;
+using Microsoft.VisualBasic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace JBlam.HarClient.Tests.Validation
+{
+    static class HarAssert
+    {
+        public static void IsValid(Har har) =>
+            IsValidLog(JAssert.HasObjectProperty(har.ToJObject(), "log"));
+        static JObject ToJObject(this object o) =>
+            JObject.FromObject(o, new JsonSerializer
```

Comment:
Look in to whether we can more tightly couple the property assignments here with the HarSerializerSettings

Reply:
No good built-in way. We _could_ set every single property, but that's problematic and tedious.

Have added a comment, so it should be visible in code review if the library changes but the test project doesn't.